scripts/install/setup_properties.sh

```diff
@@ -46,7 +46,9 @@ else
       SERVICE_ACCOUNT_NAME=$(echo $EXISTING_SA_EMAIL | cut -d @ -f 1)
     fi
   fi
-
+  # Setting the default network & subnet for Build
```

Comment:
I'm not sure what 'Build' is in this context. I think we can drop this comment entirely.

Reply:
dropped comment